CMakeLists.txt

```diff
@@ -83,7 +84,14 @@ if(CODE_COVERAGE AND NOT MSVC)
 endif()
 
 add_subdirectory(src)
-add_subdirectory(samples)
-add_subdirectory(test)
-add_subdirectory(extras)
 add_subdirectory(bindings)
+
+if(MANIFOLD_EXPORT)
+  add_subdirectory(meshIO)
```

Comment:
I'm still a bit concerned about promoting meshIO out of the test directory. I really can't stand behind it - export is decent, but import is very shaky. Formats are a lot of work - we need to careful about the promises we make. What do you think our goal should be here?

Reply:
Yeah, that's a good point, particularly about driving users into the arms of STL. @hrgdavor I'd say 3MF is probably the best, though I'm considering an extension for glTF that would make it equally good. Maybe what this really means is we should put enough effort into `meshIO` that it does a decent job. 

Let's go ahead with this change and take it as an action to improve the I/O.